OpenRA.Game/Settings.cs

```diff
@@ -284,6 +284,11 @@ public class KeySettings
 		public Hotkey ReplaySpeedFastKey = new Hotkey(Keycode.F7, Modifiers.None);
 		public Hotkey ReplaySpeedMaxKey = new Hotkey(Keycode.F8, Modifiers.None);
 
+		public Hotkey NextTrack = new Hotkey(Keycode.AUDIONEXT, Modifiers.None);
```

Comment:
My laptop only works by using a `Fn` button combo for media keys and that `Fn` key is not understood:

> The key you just pressed is not recognized by SDL. To help get this fixed, please report this to the SDL mailing list <sdl@libsdl.org> X11 KeyCode 151 (143), X11 KeySym 0x1008FF2B (XF86WakeUp).

but that is clearly an SDL upstream problem.